composer.json

```diff
@@ -15,7 +15,7 @@
     }
   ],
   "require": {
-    "php": ">=5.4.0",
+    "php": ">=5.5.0",
```

Comment:
5.5 is not supported anymore by PHP community, 5.6 has 9 months (31 Dec 2018) till end of life